### PR TITLE
task 6 endpoints and tests

### PR DESCRIPTION
**Pull request by BTMurph0**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -61,66 +62,119 @@ describe("GET /api/articles/:article_id", () => {
...
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad Request");
+      });
+  });
+  test("GET:200 responds with an appropriate  message when given an article id for an article which does not have any comments", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.msg).toBe("no comments for this article");
```
Comment (nc-kev-morel) [inline __tests__/app.test.js:177]: Nearly there, but in this instance we would need to test that an empty array is returned also
